pkg/p2p/libp2p/connections_test.go

```diff
@@ -963,22 +963,21 @@ func TestWithBlocklistStreams(t *testing.T) {
 }
 
 func TestUserAgentLogging(t *testing.T) {
-
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
-	// use concurrent-safe buffers as handlers are logging concurrently
-	s1Logs := new(buffer)
-	s2Logs := new(buffer)
+	// use concurrent-safe buffer as handlers are logging concurrently
+	logSink := new(buffer)
+	logger := logging.New(logSink, 5)
```

Comment:
It is important that two services do not share the logger in order to evaluate which one has the `(inbound)` and which one `(outbound)` log message.

Reply:
Fixed.